src/main/java/app/studnicki/ox/Menu.java

```diff
@@ -0,0 +1,69 @@
+package app.studnicki.ox;
+
+import app.studnicki.ox.config.Config;
+import app.studnicki.ox.game.Game;
+import app.studnicki.ox.game.GameBuilder;
+import app.studnicki.ox.initstate.InitState;
+import app.studnicki.ox.initstate.InitStateFactory;
+import app.studnicki.ox.ui.UserInterface;
+
+class Menu {
+
+  private final UserInterface ui;
+  private boolean exit;
+
+  Menu(UserInterface ui) {
+    this.ui = ui;
+  }
+
+  void run() {
+    while (!exit) {
+      ui.welcome();
+
+      int choice = ui.menu();
```

Comment:
is there a need for this variable? If it's only used in switch, why not `switch(ui.menu())`?